website/content/docs/release-notes/consul/v1_15_x.mdx

```diff
@@ -66,7 +66,16 @@ For more detailed information, please refer to the [upgrade details page](/consu
 
 ## Known Issues
 
-The following issues are known to exist in the v1.15.0 release:
+The following issues are known to exist in the v1.15 release:
+
+- All current 1.15.x versions are under investigation for a not-consistently-reproducible
+  issue that can cause some service instances to lose their ability to communicate in the mesh after
+  [72 hours (LeafCertTTL)](/consul/docs/connect/ca/consul#leafcertttl)
+  due to a problem with leaf certificate rotation.
```

Comment:
`LeafCertTTL` is the configuration setting that causes the problem to potentially occur at 72 hours, since 72 hours is the default value for that setting. Therefore, the original wording is more accurate, though it likely could be made clearer still!